src/web/components/ClientSideTokenGeneration/CstgDomainInputRow.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import { composeStories } from '@storybook/testing-react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import * as stories from './CstgDomainInputRow.stories';
+
+const { Default } = composeStories(stories);
+
+const validationInlineBanner = async (messageTestId: string) => {
+  expect(await screen.findByTestId(messageTestId)).toBeInTheDocument();
```

Comment:
I'm a bit concerned that we have to rely on test ids so much for our testing, given it is the last resort for [testing priorities](https://testing-library.com/docs/queries/about/#priority). 

Is there a way we can alter the component to reduce our reliance on test ids? Happy to discuss, it's not exactly my strong point 😊

Reply:
The main reason I rely on test id rather than text is that I feel we might get a couple of rounds of text changes after this feature is realised, and it easily breaks the tests.

Reply:
I see, I guess we don't export strings currently so it would mean double the changes when updating strings.